io.github.opennumismat.OpenNumismat.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.opennumismat.OpenNumismat

```

Comment:
repo name is open-numismat https://github.com/OpenNumismat/open-numismat

id should be `io.github.opennumismat.open-numismat` then.